tools/audio/audio_tablegen.c

```diff
@@ -290,19 +304,330 @@ tablegen_soundfonts(const char *sf_hdr_out, char **soundfonts_paths, int num_sou
     return EXIT_SUCCESS;
 }
 
+/* Sequences */
+
+struct seq_order {
+    size_t num_sequences;
+    char **names;
+    char **enum_names;
+    bool *ptr_flags;
+    void *buf_ptr;
+};
+
+static void
+read_seq_order(struct seq_order *order, const char *path)
```

Comment:
this hilariously long function for what is effectively a single regex capture kinda hurts my soul :)